src/style.css

```diff
@@ -1,3 +1,65 @@
+header {
+  --main-color:  #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Move font-family and default font-styles to html selector

Reply:
<img width="621" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/979d255e-a808-4d0a-adc5-215d43c120b7">